magda-typescript-common/src/generated/registry/api.ts

```diff
@@ -1,19 +1,19 @@
 /**
```

Comment:
Looks like prettier failed to run on this one.

Reply:
Have to use WSL git. Fixed.